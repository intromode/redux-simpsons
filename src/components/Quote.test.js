import React from 'react';
import { shallow } from 'enzyme';
import SimpQuote from './Quote';

describe('App component', () => {
  it('renders a App', () => {
    const wrapper = shallow(<SimpQuote quote={{
      quote: 'shanana',
      image: 'fjsdkfjl',
      character: 'me'
    
    }} />);
    expect(wrapper).toMatchSnapshot();
  });
});
