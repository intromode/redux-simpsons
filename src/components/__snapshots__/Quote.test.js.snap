// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component renders a App 1`] = `
<Fragment>
  <img
    src="fjsdkfjl"
  />
  <p>
    me
  </p>
  <p>
    shanana
  </p>
</Fragment>
`;
